src/auth/auth.module.ts

```diff
@@ -0,0 +1,25 @@
+import {forwardRef, Module} from '@nestjs/common';
+import {AuthService} from './auth.service';
+import {PassportModule} from '@nestjs/passport';
+import {JwtModule} from '@nestjs/jwt';
+import {jwtConstants} from './constant';
+import {JwtStrategy} from './jwt.strategy';
+import {TypeOrmModule} from '@nestjs/typeorm';
+import {User} from '../user/entities/user.entity';
+import {LocalStrategy} from './local.strategy';
+import {UserModule} from '../user/user.module';
+
+@Module({
+  imports: [
+    PassportModule,
+    TypeOrmModule.forFeature([User]),
+    JwtModule.register({
+      secret: jwtConstants.secret,
```

Comment:
```suggestion
      secret: JwtConstants.SECRET,
```
nest.js docs에는 `jwtConstants.secret`이 맞지만 저희가 쓰기로한 Google style guide에서는 다음과 같이 쓰는게 맞는 네이밍 컨벤션 같습니다!
ref: https://google.github.io/styleguide/jsguide.html#naming-constant-names